cli/.js/src/main/scala/aqua/builder/FinisherBuilder.scala

```diff
@@ -0,0 +1,43 @@
+package aqua.builder
+
+import aqua.io.OutputPrinter
+import aqua.js.{CallJsFunction, FluencePeer}
+import aqua.model.func.Call
+import aqua.model.func.raw.CallServiceTag
+import aqua.model.{LiteralModel, VarModel}
+
+import scala.concurrent.Promise
+import scala.scalajs.js.JSON
+
+// Will finish promise on service call
+class FinisherBuilder private (
+  serviceId: String,
+  fnName: String,
+  val promise: Promise[Unit]
+) {
+
+  def getCallServiceTag(): CallServiceTag = {
```

Comment:
I don't see a reason to have this java-style getter instead of `def callServiceTag` or even `val callTag`